### PR TITLE
Refactoring: remove utils from global scope

### PR DESCRIPTION
**Pull request by afrokick**
Based on https://github.com/monti-apm/monti-apm-agent/pull/60

### CODE REVIEW

**Comment by afrokick** [comment]
@zodern could you please re-run tests?

File: lib/client/kadira.js [hunk 1]
```diff
@@ -55,7 +56,9 @@ if (Meteor.isCordova) {
 
 let initialized = false;
 function initialize (options = {}) {
-  if (initialized) { return; }
+  if (initialized) {
```
[the remark this answers is not in the document]
Comment (afrokick) [reply lib/client/kadira.js:49]: There is a separate PR with a fix. https://github.com/monti-apm/monti-apm-agent/pull/62

**Comment by afrokick** [comment]
I found an issue with pre 1.6.0.1: for nested imports, it doesn't compile file for non-target arch.

In our case, we import httpRequest from client/utils, but tests were compiled for os.

I just move httpRequest to separate file and assign it to global Kadira._makeHttpRequest. In future, we need a more elegant solution.

**Comment by afrokick** [comment]
@leonardoventurini ready.

**Comment by zodern** [comment]
Thanks @afrokick!

**Pull request merged**